Bindings/simulation.i

```diff
@@ -229,3 +229,24 @@
 %template(getMuscleList) OpenSim::Model::getComponentList<OpenSim::Muscle>;
 %template(getModelComponentList) OpenSim::Model::getComponentList<OpenSim::ModelComponent>;
 %template(getJointList) OpenSim::Model::getComponentList<OpenSim::Joint>;
+
+%include <OpenSim/Actuators/osimActuatorsDLL.h>
+%include <OpenSim/Actuators/MuscleFirstOrderActivationDynamicModel.h>
+%include <OpenSim/Actuators/MuscleFixedWidthPennationModel.h>
+%include <OpenSim/Actuators/Thelen2003Muscle.h>
+%include <OpenSim/Actuators/Millard2012EquilibriumMuscle.h>
+
+%template(Thelen2003MuscleList)
+    OpenSim::ComponentList<const OpenSim::Thelen2003Muscle>;
+%template(Thelen2003MuscleIterator)
+    OpenSim::ComponentListIterator<const OpenSim::Thelen2003Muscle>;
+
+%template(Millard2012EquilibriumMuscleList)
+    OpenSim::ComponentList<const OpenSim::Millard2012EquilibriumMuscle>;
+%template(Millard2012EquilibriumMuscleIterator)
+    OpenSim::ComponentListIterator<const OpenSim::Millard2012EquilibriumMuscle>;
+
+%template(getThelen2003MuscleList)
+  OpenSim::Model::getComponentList<Thelen2003Muscle>;
```

Comment:
Probably doesn't matter, but based on the rest of this file, `Thelen2003Muscle` should be `OpenSim::Thelen2003Muscle`.